### PR TITLE
[DDS-1057] ElasticSearch 8.5 image

### PR DESCRIPTION
**Pull request by santhureturi308**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

Explain the **details** for making this change. What existing problem does the pull request solve?

# Changelog Entry

# Closing issues
https://digital-vic.atlassian.net/browse/DDS-1057

### CODE REVIEW

**Review by nicksantamaria — CHANGES_REQUESTED** [review]
Looks good @santhureturi308 

Before merging could you also delete all the other files in this directory?

[build.gradle](https://github.com/dpc-sdp/bay/blob/DDS-105-adding-Elasticsearch-8.5-image-to-Bay/images/bay-elasticsearch/build.gradle)
[gradle.properties](https://github.com/dpc-sdp/bay/blob/DDS-105-adding-Elasticsearch-8.5-image-to-Bay/images/bay-elasticsearch/gradle.properties)
[readonlyrest.yml](https://github.com/dpc-sdp/bay/blob/DDS-105-adding-Elasticsearch-8.5-image-to-Bay/images/bay-elasticsearch/readonlyrest.yml)
[settings.gradle](https://github.com/dpc-sdp/bay/blob/DDS-105-adding-Elasticsearch-8.5-image-to-Bay/images/bay-elasticsearch/settings.gradle)

File: images/bay-elasticsearch/Dockerfile [hunk 1]
```diff
@@ -1,30 +1,3 @@
...
-RUN bin/elasticsearch-plugin install file:///tmp/readonlyrest/es61x/build/distributions/readonlyrest-1.16.21_es6.2.4.zip --batch \
-    && chown elasticsearch:root config/readonlyrest.yml \
-    && chown -R elasticsearch:root plugins/readonlyrest \
-    && yum erase java-1.8.0-openjdk-devel -y \
-    && yum erase unzip -y\
-    && rm -f /tmp/v1.16.21_es6.2.4.zip \
-    && rm -rf /tmp/readonlyrest
+FROM elasticsearch:8.5.3
 
+RUN /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch ingest-attachment
```
Comment (nicksantamaria) [inline images/bay-elasticsearch/Dockerfile]: Could you install these oplugins too
-    "analysis-kuromoji"
-    "analysis-icu"

**Comment by krakerag** [comment]
@nicksantamaria @santhureturi308 Can I access this image before we merge this to run up a local build?

**Comment by nicksantamaria** [comment]
@krakerag yep I'll push a tag to this branch's HEAD which will push the artefacts to dockerhub

**Comment by krakerag** [comment]
> @krakerag yep I'll push a tag to this branch's HEAD which will push the artefacts to dockerhub

Give me a yell in slack when I can give it a run ;)

**Comment by nicksantamaria** [comment]
@krakerag image available for testing - https://hub.docker.com/repository/docker/singledigital/bay-elasticsearch/tags?page=1&ordering=last_updated

File: images/bay-elasticsearch/Dockerfile (hunk 1 above)
Comment (santhureturi308) [reply images/bay-elasticsearch/Dockerfile]: @nicksantamaria Thanks for checking; I have updated the docker file to pick the advice plugins.

**Comment by krakerag** [comment]
I got this to build with the following dockerfile

```
FROM singledigital/bay-elasticsearch:pr-DDS-1057
ARG ES_TPL

ENV ES_TPL=${ES_TPL:-elasticsearch.yml}

COPY .docker/elasticsearch.yml .docker/elasticsearch.* /tmp/elasticsearch/

RUN mkdir -p config \
    && cp /tmp/elasticsearch/$ES_TPL config/elasticsearch.yml

ENV ES_JAVA_OPTS "-Xms1000m -Xmx1000m"
```

Drupal reports the following which is fine:

image

I've done this with a colima stack instead of a docker desktop for mac stack, it would be good if @GROwen or someone else that is still using docker desktop can test this also.

**Review by nicksantamaria — APPROVED** [review]

**Pull request merged**